www/generate_benchmark_list.js

```diff
@@ -0,0 +1,80 @@
+// Copyright 2016 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+// Author: William Haack (will@cockroachlabs.com)
+
+function loadTestListFromJSON(callback) {
+    loadJSON("test_names.json", callback);
+}
+
+function getTests(directory) {
+    var tests = []
+    for (var testIndex in directory) {
+	tests.push(directory[testIndex])
+    }
+    return tests
+}
+
+function generateTestListHTML(directory, tests) {
+    text = '';
+    for (var testIndex in tests) {
+	console.log(directory);
+	text += '&nbsp;&nbsp;&nbsp;&nbsp;<a href="/plot.html?directory=' + directory + '&test=' +
```

Comment:
If you're just using the non-breaking spaces for alignment, you could probably do this with CSS instead.